lib/TAP/Formatter/Color.pm

```diff
@@ -12,7 +12,7 @@ my $NO_COLOR;
 BEGIN {
     $NO_COLOR = 0;
 
-    eval 'use Term::ANSIColor';
+    eval 'use Term::ANSIColor qw//';
```

Comment:
> Changing code to make dzil happier sounds wrong

Oh yes, I totally agree -- I was just describing what the issue is so one can watch out for it.